### ISSUE TITLE
Update readme to point to new maintained AEM groovy console 

### ISSUE THREAD

**Issue by royteeuwen** [CID15/aem-groovy-console#14]
We are currently maintaining AEM Groovy Console and adding features / making sure it's compatible with the Cloud.

Could you update the link to point to the following repo:
https://github.com/orbinson/aem-groovy-console

**Comment by royteeuwen** [comment]
@markdaugherty  that would just make ACS AEM Commons even a bigger monolith and taking away all the flexibility that you have by making separate frameworks :/. There is already an active community working on the AEM Groovy Console, where we are taking the security concerns into account. We have upgraded to Java 17, made it AEM cloud compatible, updated all the repo structure to the new standards. You'd just have two versions living at the same time, without any added benefit? ACS AEM Commons isn't officially supported by Adobe either, I have done multiple contributions to it, and it's also community based

**Comment by kwin** [comment]
Strongly agree with what @royteeuwen said above. I think this tool is so big it should be installable standalone. It doesn't really matter which Maven coordinates it has, let us just join forces here instead of maintaining different forks.

**Comment by royteeuwen** [comment]
Indeed, I'd love to give you committer rights at the Orbinson groovyconsole so we can work together on it @markdaugherty, not here trying to steal something. We just took over during your absence so that this wonderful project could still live and thrive!

**Comment by tomasz-sobczyk-wttech** [comment]
I would like to back up what @kwin  / @royteeuwen is talking about. Groovy Console is vital for multiple large-scale AEM platforms. In the last few years ecosystem has moved towards the version maintained here:
https://github.com/orbinson/aem-groovy-console
It would be really an awful shame if we had two competing versions of it. Folks from Orbinson really saved a lot of people here by picking up the maintenance of it and I personally believe they earned their stripes to be maintainers of a joint version.
Keeping the development within the orbinson org would reduce the impact on community at this stage.

**Comment by markdaugherty** [comment]
I would prefer to transfer the repo from CID15 to orbinson rather than update the `README`.  However, I'm not sure the best way to do this given that the repo has the same name in both orgs.  Here is one idea (maybe too complicated?)...

1. Merge everything from orbinson repo to CID15 repo before transfer
2. Temporarily delete the orbinson repo
3. Transfer repo from CID15 to orbinson

Thoughts?

Further complicating the question is that the old ICF repo is still out there.  The `README` informs that it is no longer maintained, but ideally we would transfer this repo as well.  I am reaching out to some folks there (the org has changed entirely and I'm not sure if anyone even knows about those GitHub projects) to see what we can do.

**Comment by royteeuwen** [comment]
@markdaugherty  they say seeing as we aren't real forks, the only way is to indeed use the archiving functionality and update the readme to point to each other.

I find your solution of the temporary deleting the Orbinson and then transferring this one definitely interesting. Any way we could test that it works though, did you find a wiki page about it somewhere?

**Comment by markdaugherty** [comment]
I have not seen any documentation on the approach, it's entirely theoretical.  :-)

I am still waiting on a follow-up from someone at ICF regarding that repo, although it's looking doubtful that we'll be able to get that out from under them.
